doc/adrs/122_add_support_for_board_representation.adoc

```diff
@@ -0,0 +1,110 @@
+
+=== Edit mode
+
+* In this mode, it's possible to change the position and the size of all the representations.
+New representations can also be added by dragging them from the explorer.
+* The representations can be reorganized by drag and drop action.
+All the representations can be resized via a handle placed in the bottom right corner of the representation.
+If the resize of a representation overlaps another one, this second representation is moved away.
+* A board-type representation, has no horizontal or vertical boundaries.
+In case of overflow, there will be scrollbars for navigating.
+If a user doesn't want the page to be scrolled, it is possible by adjusting the layout.
+* Only in this mode, a representation can be deleted by using the _del_ key.
+This deletion does not affect the layout of the other representations.
+* When editing the _board_, the representations are read-only.
+* Since a board representation is itself a representation, it can be added to another _board_.
+There is limitation to avoid a _board_ including another _board_ being added to this second _board_.
+It prevents a _board_ loop.
+
+=== Direct mode
+
+* In this mode, the representations can't be moved or resized.
+* Representations on a _board_ have exactly the same behavior as if they were opened individually.
+A user can thus modify them.
+* If the representation provides one, the user has access to its palette.
+
+image:images/122/122_board_representation_example.png[board_representation_example]
+
+* If the defined size of a representation does not allow it to be displayed in its entirety, a scroll bar will be added in it.
+For diagram-type representations, it is possible to zoom inside.
+* Selecting an element from the explorer or from one of the _board_ representations changes the selection in all representations.
+* If a representation is deleted but always referenced in a _board_, its location is replaced by a blank slot with a message indication that the representation doesn't exist
+anymore.
+
+=== Representation header
+
+* Each representation on the _board_ has a header on top of it, with the breadcrumb representing their path relative to the semantic element to which the board representation is
+attached.
+
+image:images/122/122_representation_gantt_with_header.png[representation_with_header]
+
+=== Board palette
+
+image:images/122/122_board_palette.png[board_palette]
+
+* The board palette is always visible in direct and edit mode in the top left of the main zone.
+* The board palette is composed of four buttons:
+
+. toggle to full screen
+. share button (opens a popup with the URL to access to the _board_ representation)
+. toggle to edit mode
+. toggle to direct mode
+
+=== Representations panel
+
+* The right panel of the sirius page includes a representation view.
+* For _board_, this view presents, in the form of a tree, the representations it contains.
+* For each of these representations, there is a trash icon.
```

Comment:
please describe what does the trash icon. If I understand the following sentence, delete on representation will delete the representation itself whereas delete on representation inside a board wil only remove the reference from the board to the representation.
And by the way I consider it is confusing.

So at least do the proposed change proposed by sbegaudeau(have different icons)